ocs_ci/helpers/helpers.py

```diff
@@ -4681,10 +4681,13 @@ def retrieve_cli_binary(cli_type="mcg"):
 
     remote_path = get_architecture_path(cli_type)
     remote_cli_basename = os.path.basename(remote_path)
-    if cli_type == "mcg":
+    if cli_type == "mcg" and semantic_version < version.VERSION_4_20:
         local_cli_path = constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH
     elif cli_type == "odf":
         local_cli_path = os.path.join(config.RUN["bin_dir"], "odf-cli")
+    elif cli_type == "mcg" and semantic_version >= version.VERSION_4_20:
+        local_cli_path = os.path.join(config.RUN["bin_dir"], "odf-cli noobaa")
```

Comment:
I think this will be a problem here as you have a space `odf-cli noobaa`

This is mainly used to extract the binary - and later on in line 4707 we. do rename of that locsl_cli_path , which will not work as expected here.

We just need to update above condition I think to something like:
```
elif cli_type == "odf" or cli_type == "mcg" and semantic_version >= version.VERSION_4_20:
```
And this new elif block can be deleted as we will just have `odf-cli`.